lesson34/src/mypage.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="ja">
```

Comment:
nits
Since the content is not in ja, it would be better in terms of accessibility to specify lang individually or to align all the content in Japanese.

Reply:
Thanks for the review. The article has been corrected to Japanese.
[7096769](https://github.com/sae-github/JS_lesson/pull/43/commits/70967696234eec82b01ba3e90662d9cb9421ceb8)